.github/CONTRIBUTING.md

```diff
@@ -6,28 +6,31 @@ Below you'll find guidelines for contributing that will keep our codebase clean
 
 ## Table of Contents
 
-* [How can I contribute?](#how-can-i-contribute)
-    * [Bug reports](#bug-reports)
-    * [Resolving issues](#resolving-issues)
-    * [Making your first contribution](#making-your-first-contribution)
-        * [Setting up the environment](#setting-up-the-environment)
-        * [Writing tests](#writing-tests)
-            * [API Coverage Tests](#api-coverage-tests)
-            * [Engine tests](#engine-tests)
-        * [Running tests / coverage reports](#running-tests--coverage-reports)
-        * [Making a Pull Request](#making-a-pull-request)
-* [Code style guidelines](#code-style-guidelines)
-    * [Foolish consistency](#foolish-consistency)
-    * [Method docstrings](#method-docstrings)
-        * [Descriptions](#descriptions)
-        * [Links to related API endpoints](#links-to-related-api-endpoints)
-        * [Parameters](#parameters)
-        * [Returns](#returns)
-        * [Docstring examples](#docstring-examples)
+* [Contributing to CanvasAPI](#contributing-to-canvasapi)
+    * [Table of Contents](#table-of-contents)
+    * [How can I contribute?](#how-can-i-contribute)
+        * [Bug reports](#bug-reports)
+        * [Resolving issues](#resolving-issues)
+        * [Making your first contribution](#making-your-first-contribution)
+            * [Setting up the environment](#setting-up-the-environment)
+            * [Writing tests](#writing-tests)
+                * [API coverage tests](#api-coverage-tests)
+                * [Engine tests](#engine-tests)
+            * [Running tests / coverage reports](#running-tests--coverage-reports)
+            * [Making a pull request](#making-a-pull-request)
+    * [Code style guidelines](#code-style-guidelines)
+        * [Running code style checks](#running-code-style-checks)
+        * [Foolish consistency](#foolish-consistency)
+        * [Method docstrings](#method-docstrings)
+            * [Descriptions](#descriptions)
+            * [Links to related API endpoints](#links-to-related-api-endpoints)
+            * [Parameters](#parameters)
+            * [Returns](#returns)
+            * [Docstring Examples](#docstring-examples)
 
 ## How can I contribute?
 
-### Bug Reports
+### Bug reports
 
```

Comment:
Why the removal of Title Case?

Reply:
Honestly, I think Sentence case just reads more fluidly. [Arguments to be made in both directions, though.](https://www.herosmyth.com/article/headline-capitalization-ux-title-case-vs-sentence-case)